test.el

```diff
@@ -271,14 +271,14 @@ that buffer once BODY finishes."
                 (expand-file-name "root/bazel-root/external/ws/bbb.h" dir)))))))
 
 (ert-deftest bazel-mode/fill ()
-  "Check that “keep sorted” comments are left alone."
+  "Check that magic comments are left alone."
   (with-temp-buffer
     (insert-file-contents
      (expand-file-name "testdata/fill.BUILD" bazel-test--directory))
     (bazel-mode)
-    (search-forward "# The Foobar files")
     (let ((before (buffer-string)))
-      (fill-paragraph)
+      (while (search-forward "# Test paragraph" nil t)
+        (fill-paragraph))
       (should (equal (buffer-string) before)))))
```

Comment:
Maybe move this assertion into the `while` body. The contents should stay the same after each fill operation.

Reply:
Done.  Also added a little breadcrumb to aid debugging with `ert-info`.